server/src/routes.rs

```diff
@@ -0,0 +1,38 @@
+use super::sql;
+use std::collections::HashMap;
+
+use actix_web::{get, HttpResponse};
+
+pub struct State {
+    pub db_conn: sqlx::SqlitePool,
+}
+
+type Data = actix_web::web::Data<State>;
+
+#[derive(serde::Serialize)]
+struct ErrorJsonResp<'m> {
+    msg: &'m str,
+    details: String,
+}
+
+#[get("/add")]
+pub(super) async fn add() -> HttpResponse {
+    HttpResponse::Ok().json(HashMap::from([("msg", "/add")]))
```

Comment:
Note that these routes are for maintaining marks, not assigning packages

Reply:
I will keep the same implementation of the old API.